### ISSUE TITLE
Q: Is it possible to install specific versions to a distro?

### ISSUE THREAD

**Issue by stevecs** [zabbly/zfs#5]
I didn't see any other versions in the apt list than the latest (2.2.6) for jammy (22.04) amd64.   Was looking for a repository that would allow relative ease to configure on system to specifically NOT go to  the latest but to say latest - 1 branch or latest -2.  (i.e. 2.1.15; or 2.0.7) to keep in line with our/my standard deployment practice.

**Comment by stgraber** [comment]
No, we pretty aggressively cleanup the package list in the repository (only current and previous build) as this isn't meant as an archive of all ZFS versions.

It shouldn't be particularly difficult for you to grab the two packaging commits and apply them to the version of ZFS of your choice:
 - https://github.com/zabbly/zfs/commit/5dc37b70ccc704e39ae9203e7bd26bd81cc5d4b2
 - https://github.com/zabbly/zfs/commit/67b097057f3645bec66b2e447b481ff0f7cf504b

**Comment by stevecs** [comment]
Thanks.  I'll look elsewhere for a repository then or perhaps need to create a local one.
